docker/release_dockerfile

```diff
@@ -13,6 +13,6 @@ COPY release.sh ./release.sh
 
 RUN chmod a+x ./release.sh
 
-VOLUME ["/tikv"]
+WORKDIR /tikv
```

Comment:
Because this docker is only for compilation. The compiled result may be move to another docker or uploaded to another machine.